rvic/core/read_forcing.py

```diff
@@ -196,7 +196,7 @@ def start(self, timestamp, rout_var):
             if units in ['kg/m2*s', 'kg m-2 s-1', 'kg m^-2 s^-1',
                          'kg*m-2*s-1', 'kg s-1 m-2']:
                 self.fld_mult[fld] = 1.0
-            elif units in ['mm', 'MM', 'milimeters', 'Milimeters']:
+            elif units in ['mm', 'MM', 'milimeters', 'Milimeters', 'mm.d-1']:
```

Comment:
MILLIMETERS has two 'll's: millimeter

Also, these units are not equivalent. The first are 4 are a depth (presumably per timestep), while the latter is a rate with a hard-wired timestep of d-1. This is confusing. You should cover the rate separately or use something like udunits  (there is a python package cfunits - https://code.google.com/p/cfunits-python/) to handle different input units.